cookbook/plans/default.json

```diff
@@ -39,7 +39,12 @@
                     "geolocation_profile": "18446744073709551615",
                     "total_cu_limit": "1000000",
                     "epoch_cu_limit": "100000",
-                    "max_providers_to_pair": "3"
+                    "max_providers_to_pair": "3",
+                    "selected_providers_mode": 1,
+                    "selected_providers": [
+                        "lava@1w3kctt4fzy3u6datmym0hl2d4uhxtat3ut2eqq",

```

Comment:
this cant be here, those addresses are used generated on your local machine

Reply:
done